### PR TITLE
[GR-1383] Loading dnaSeqQC from Dashi.

### PR DESCRIPTION
**Pull request by slazicoicr**
dnaSeqQC will replace bamqc4.

### CODE REVIEW

File: application/dash_application/utility/df_manipulation.py [hunk 1]
```diff
@@ -290,6 +291,17 @@ def get_bamqc3_and_4():
     )
 
 
+def get_dnaseqqc_and_bamqc4():
```
Comment (callunity) [inline application/dash_application/utility/df_manipulation.py:294]: Is calling the function outside of the scope of the ticket?

**Review by avarsava — APPROVED** [review]

File: application/dash_application/utility/df_manipulation.py (hunk 1 above)
Comment (slazicoicr) [reply application/dash_application/utility/df_manipulation.py:294]: I read it as such. The ticket was more for getting dnaSeqQC into Dashi for future use. Right now, there are no actions for it in [Shesmu](https://shesmu.gsi.oicr.on.ca/olivedash?alert=W10&saved=eyJmaWxlIjoiL2hvbWUvZGViaWFuL3NwYi1zZXF3YXJlLXByb2R1Y3Rpb24vc2hlc211L3Byb2R1Y3Rpb24vbmlhc3NhLWRuYVNlcVFDLnNoZXNtdSIsImxpbmUiOjE2LCJjb2x1bW4iOjYsImhhc2giOiJGRDg3RTE2NjBENTFCNTBCMDU1MEFGNTc4NzdFMDVBM0RCOEMwRUNGIn0&filters=e30)

Comment (callunity) [reply application/dash_application/utility/df_manipulation.py:294]: Aha, I understand. dnaSeqQC will be run in [shesmu-research](https://shesmu-research.gsi.oicr.on.ca/actiondash?filters=eyJzb3VyY2VmaWxlIjpbIi9ob21lL2RlYmlhbi9zcGItc2Vxd2FyZS1wcm9kdWN0aW9uL3NoZXNtdS9yZXNlYXJjaC92aWRhcnItZG5hc2VxcWMuc2hlc211Il19&saved=IkFsbCBBY3Rpb25zIg), and there are some actions that are already running there.

Comment (callunity) [reply application/dash_application/utility/df_manipulation.py:294]: Given that, could this PR please be updated to make the changes in Dashi that are required for the cache to actually be used?

**Review by callunity — APPROVED** [review]
Thanks Savo!

**Pull request merged**